packages/flutter/lib/src/widgets/focus_scope.dart

```diff
@@ -401,27 +401,29 @@ class Focus extends StatefulWidget {
   /// {@endtemplate}
   final bool canRequestFocus;
 
-  /// {@template flutter.widgets.Focus.overrideFocusability}
-  /// If true, setting [canRequestFocus] on this widget to false will disable
-  /// focus for its descendants.
+  /// {@template flutter.widgets.Focus.descendantsAreFocusable}
+  /// If false, will make this widget's focus node and its descendants
```

Comment:
If I understand correctly, this should only mention its descendants. Itself is unaffected

Reply:
Correct. Fixed.